README.md

```diff
@@ -25,11 +25,6 @@ Wasm provides a number of benefits outside of the web including:
 	Wasm is a high-level assembly, independent from any specific hardware.
 	This has the potential to provide a universal specification of computation that can execute in the cloud, on the edge, or in an embedded device.
 
-**A note on naming.**
```

Comment:
I use this on my resume. Can I ask this not be retained for selfish reasons?

Reply:
Yeah, we should retain this comment.

We can remove the "lingering references" and "patience" comment.

Reply:
I attempted a rewrite of the acknowledgements section to capture these contributions. Happy to credit Gregor, but I'm not clear why we need to have a note on the silverfish name after all references have been wiped, since it's largely historical at that point.

Reply:
I think it’s generally good practice when renaming something to leave a note of the previous name.

For example I’ve linked to this repo and mentioned the project name externally before, and I don’t really want to go track down those references and change the name I used.

Any harm in adding a “previously known as” or “Gregor worked on this under the name ...” sentence?

Reply:
No harm. We should do that.

Reply:
Added a "previously known as" note under the title.